conf/spark-env.sh

```diff
@@ -43,8 +42,6 @@ export LIBPROCESS_SSL_KEY_FILE=/mnt/mesos/sandbox/.ssl/scheduler.key
 export MESOS_MODULES="{\"libraries\": [{\"file\": \"libdcos_security.so\", \"modules\": [{\"name\": \"com_mesosphere_dcos_ClassicRPCAuthenticatee\"}]}]}"
 export MESOS_AUTHENTICATEE="com_mesosphere_dcos_ClassicRPCAuthenticatee"
 
-echo "spark-env: Environment" >&2
-env >&2
```

Comment:
Why did you remove this?

Reply:
I added that when I was debugging strict mode. I don't think a customer would necessarily want to spew the environment to the log every job.